package.json

```diff
@@ -62,45 +62,44 @@
     "passport": "^0.4.0",
     "passport-custom": "^1.1.0",
     "passport-google-oauth20": "^2.0.0",
-    "query-string": "^6.8.2",
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0",
-    "timeago.js": "^4.0.0-beta.2"
+    "query-string": "^6.8.3",
+    "react": "^16.11.0",
+    "react-dom": "^16.11.0",
+    "timeago.js": "^4.0.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.5.5",
+    "@babel/core": "^7.6.4",
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
-    "@babel/plugin-transform-modules-commonjs": "^7.5.0",
-    "@babel/preset-env": "^7.5.5",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/preset-typescript": "^7.3.3",
-    "@zeit/ncc": "^0.20.4",
+    "@babel/plugin-transform-modules-commonjs": "^7.6.0",
+    "@babel/preset-env": "^7.6.3",
+    "@babel/preset-react": "^7.6.3",
+    "@babel/preset-typescript": "^7.6.0",
     "babel-eslint": "^11.0.0-beta.0",
     "babel-jest": "^24.9.0",
     "babel-loader": "^8.0.6",
     "body-parser": "^1.19.0",
     "clean-webpack-plugin": "^3.0.0",
     "concurrently": "^5.0.0",
     "copyfiles": "^2.1.1",
-    "cross-env": "^5.2.0",
+    "cross-env": "^6.0.3",
     "db-migrate": "^0.11.6",
     "db-migrate-mysql": "^1.1.10",
-    "eslint": "^6.3.0",
-    "eslint-config-ts-react-important-stuff": "^1.0.0",
+    "eslint": "^6.6.0",
+    "eslint-config-ts-react-important-stuff": "^1.1.0",
     "file-loader": "^4.2.0",
-    "fork-ts-checker-webpack-plugin": "^1.5.0",
-    "husky": "^3.0.4",
+    "fork-ts-checker-webpack-plugin": "^3.0.1",
```

Comment:
https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/releases/tag/v2.0.0 and https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/releases/tag/v3.0.0